controllers/platform/services.go

```diff
@@ -99,27 +98,9 @@ func createDataIndexDeployment(ctx context.Context, client client.Client, platfo
 	liveProbe := readyProbe.DeepCopy()
 	liveProbe.ProbeHandler.HTTPGet.Path = common.QuarkusHealthPathLive
 	dataDeployContainer := &corev1.Container{
-		Image: common.DataIndexImageBase + common.PersistenceTypeEphemeral,
-		Env: []corev1.EnvVar{
-			{
-				Name:  "KOGITO_DATA_INDEX_QUARKUS_PROFILE",
-				Value: "http-events-support",
-			},
-			{
-				Name:  "QUARKUS_HTTP_CORS",
-				Value: "true",
-			},
-			{
-				Name:  "QUARKUS_HTTP_CORS_ORIGINS",
-				Value: "/.*/",
-			},
-		},
-		Resources: corev1.ResourceRequirements{
-			Limits: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("100m"),
-				corev1.ResourceMemory: resource.MustParse("256Mi"),
-			},
-		},
+		Image:          svc.GetServiceImageName(common.PersistenceTypeEphemeral),
+		Env:            svc.GetEnvironmentVariables(),
+		Resources:      svc.GetPodResourceRequirements(),
```

Comment:
`svc` is an alias for Kubernetes Services, can we change this since this context is different, please?

Reply:
per golang convention, this can be simply `ps`  (shorten for PlatformService)

Reply:
Done.